bin/alignment.py

```diff
@@ -196,16 +196,17 @@ def processData(json_obj):
         samples_data)
     output_json = json.dumps(output_dict)
     return output_json
+# read input till the EOF
+for line in iter(sys.stdin.readline,''):
+    input_json += line
 
+# process data
+json_obj = json.loads(input_json)
+processedData = processData(json_obj)
```

Comment:
before passing 'json_obj' to 'processData'  should it not be checked whether it is a valid json or not ? In case if the 'json' is malformed, 'processData()' will complain about it and eventually fail to execute.

Reply:
@rish9511 Qt handles all this- if python is failing in any case, it will not send complete data and Qt will try to make a json then it'll handle all kind of this error.